### PR TITLE
[PLATFORM-3432] Update fork with upstream (v1.1.3)

### PR DESCRIPTION
**Pull request by Evanjt1**
PLATFORM-3432
---

Latest version of upstream (v1.1.3) has some updates to the login sync api that we'll need for some upcoming features. This PR brings our fork up to current head of upstream.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,10 @@
   },
   "scripts": {
     "start": "node dist/server.js",
+    "debug": "node --inspect dist/server.js",
     "build": "rimraf dist && tsc -p tsconfig.prod.json",
     "release": "npm run build && ./scripts/release.sh",
-    "playwright": "npm run build && docker-compose -f docker-compose.playwright.yml up --build -d && cd playwright && playwright test && docker-compose -f ../docker-compose.playwright.yml down",
+    "playwright": "npm run build && docker compose -f docker-compose.playwright.yml up --build -d && cd playwright && playwright test && docker compose -f ../docker-compose.playwright.yml down",
```
Comment (aflesher) [inline package.json:17]: Is this a recent change to `docker`? I've always used `docker-compose`

File: sequence/logout.puml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+break 
+ sdk -> as: client_id=sdk_version\n&refresh_token=refresh_token
+ activate sdk
+    activate as
+  as -> db: Find User
+  db --> as: User Data\nlogout later than token
+  as --> sdk: {error=access_denied}
+    deactivate as
+deactivate sdk
+@enduml
```
Comment (aflesher) [inline sequence/logout.puml:36]: Great to see this plantuml diagrams added to the source. Not a blocker but we should add an image of the rendered diagrams to the source like our other repos.

**Review by aflesher — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (Evanjt1) [reply package.json:17]: @aflesher https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

**Pull request merged**